lib/jsdom/living/events/PopStateEvent.webidl

```diff
@@ -1,6 +1,7 @@
-[Exposed=Window,
- Constructor(DOMString type, optional PopStateEventInit eventInitDict)]
+// https://html.spec.whatwg.org/multipage/browsing-the-web.html#the-popstateevent-interface
+[Exposed=Window]
 interface PopStateEvent : Event {
+  constructor(DOMString type, optional PopStateEventInit eventInitDict = {});
```

Comment:
[ditto](https://html.spec.whatwg.org/multipage/browsing-the-web.html#the-popstateevent-interface)